From: NARUSE, Yui <naruse@users.sourceforge.jp>
Date: Wed, 15 Sep 2010 00:04:17 +0000 (+0900)
Subject: Fix test: test_data/ruby-dev:39722.
X-Git-Url: http://git.sourceforge.jp/view?p=nkf%2Fnkf.git;a=commitdiff_plain;h=bbb280a04623e073e447add83c6c9a5309c30fd2

Fix test: test_data/ruby-dev:39722.

Don't use non ASCII in nkf_test.pl.
[nkf-bug#23228]
---

diff --git a/nkf_test.pl b/nkf_test.pl
index 30b754b..6296f32 100644
--- a/nkf_test.pl
+++ b/nkf_test.pl
@@ -1003,8 +1003,8 @@ printf "%-40s", "test_data/bugs10904";
     &test("$nkf -Mj",$example{'test_data/bugs10904'},$example{'test_data/bugs10904.ans'});
 
 printf "%-40s", "test_data/ruby-dev:39722";
-    &test("$nkf -Mj",<<eom,<<eom);
-aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaあ
+    &test("$nkf -MjW",<<eom,<<eom);
+aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\xE3\x81\x82
 eom
 =?US-ASCII?Q?aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa?=
  =?US-ASCII?Q?aaaaaaaaaaaaaaaaa?= =?ISO-2022-JP?B?GyRCJCIbKEI=?=
